opencti-platform/opencti-front/src/utils/Localization.js

```diff
@@ -4566,20 +4570,20 @@ const i18n = {
       'relationship_subnarrative-of': 'サブナラティブ',
       relationship_has: '存在する',
       relationship_sample: 'サンプル',
-      'relationship_body-multipart': 'body multipart',
-      'relationship_body-raw': 'body raw',
-      relationship_child: 'child',
-      'relationship_creator-user': 'creator',
-      relationship_detects: 'detects',
-      'relationship_dst-payload': 'payload',
-      'relationship_encapsulated-by': 'encapsulated by',
-      relationship_encapsulates: 'encapsulates',
-      'relationship_opened-connection': 'opened connection',
-      'relationship_operating-system': 'operating system',
-      relationship_parent: 'parent',
-      'relationship_parent-directory': 'parent directory',
-      'relationship_raw-email': 'raw email',
-      'relationship_src-payload': 'src payload',
+      'relationship_body-multipart': 'メールマルチパートのボディ',
+      'relationship_body-raw': 'ボディデータ',
+      relationship_child: '子',
+      'relationship_creator-user': '作成者',
+      relationship_detects: '検知',
+      'relationship_dst-payload': '宛先ペイロード',
+      'relationship_encapsulated-by': 'カプセル化される',
+      relationship_encapsulates: 'カプセル化する',
+      'relationship_opened-connection': '接続されている',
+      'relationship_operating-system': 'OS',
+      relationship_parent: '親',
+      'relationship_parent-directory': '親ディレクトリ',
+      'relationship_raw-email': 'メールデータ',
+      'relationship_src-payload': '送信元のペイロード',
```

Comment:
[IMO]
relationship_dst-payloadと合わせると

```suggestion
      'relationship_src-payload': '送信元ペイロード',
```

ですかね？